src/rdkafka_cgrp.c

```diff
@@ -5144,8 +5162,25 @@ rd_kafka_cgrp_calculate_subscribe_revoking_partitions(
 static void
 rd_kafka_cgrp_subscription_set(rd_kafka_cgrp_t *rkcg,
                                rd_kafka_topic_partition_list_t *rktparlist) {
+
+        int is_wildcard_subscription_present = 0;
```

Comment:
This bool isn't necessary you can put the `rd_kafka_topic_partition_list_regex_cnt` call in the if condition and then use `rkcg->rkcg_flags & RD_KAFKA_CGRP_F_WILDCARD_SUBSCRIPTION`